js/stdlib/ts/ETH_like.ts

```diff
@@ -1176,6 +1176,7 @@ const ethLike = {
   setMinMillisBetweenRequests,
   setCustomHttpEventHandler,
   setSigningMonitor,
+  getTimeSecs,
```

Comment:
Don't you need to change `ETH` and `CFX` too? I'm pretty sure you do. You should write a test program that uses it (or stick it in an existing program)

Reply:
ah whoops. trying to move too fast today